modules/ROOT/pages/create-api-specification-studio.adoc

```diff
@@ -0,0 +1,41 @@
+= Create an API Project in Studio
+
+Anypoint Studio 7.7 and later enables you to create a RAML or OAS 2.0 and 3.0 API specification, or an API fragment in Studio and automatically synchronize it with Design Center.
+
+It is not possible to create a flow out of an OAS 3.0 API specification.
+
+[NOTE]
+Studio uses the EGit plugin only for the VCS feature of editing API specifications offline. Studio does not support the EGit plugin for Mule application projects that you might track using your own VCS.
+
+To start creating an API project in Studio, you must be logged in to Anypoint Platform. See xref:xref:set-credentials-in-studio-to.adoc[Configure Platform Credentials].
```

Comment:
Why not make this the first step?
I would omit linking here to actually creating the login....

Reply:
Also consider using the template:
== Prerequisites, if any
== Steps